search/util.py

```diff
@@ -0,0 +1,38 @@
+"""Utility functions for arxiv.search."""
```

Comment:
This innocently named file contains arxiv identifier parsing.

A very serious maintenance problem with legacy arxiv is redundently
implemented identifier handling.

This must go to some sort of shared reusable library/module etc.

Reply:
Indeed. ARXIVNG-210 should do just that

Reply:
This is done in [base-0.4](https://github.com/cul-it/arxiv-base/releases/tag/0.4), and reflected in https://github.com/cul-it/arxiv-search/pull/108